app/root.tsx

```diff
@@ -119,6 +121,8 @@ const Document = ({ children, title }: { children: React.ReactNode; title?: stri
   )
 }
 
+createEmotionCache({ key: "mantine" })
```

Comment:
should we change this to be pokt-blocks or something other than mantine?

Reply:
Sure, makes sense for me 🔥